web/app.fluidity.money/app/routes/$network/query/projectedWinnings.tsx

```diff
@@ -13,6 +14,7 @@ export const loader: LoaderFunction = async ({ params, request }) => {
 
   const url = new URL(request.url);
   const address = url.searchParams.get("address");
+  const useMoralis = useSplitExperiment("enable-moralis", true);
```

Comment:
Isn't this vaguely problematic because it'll return `undefined` here if disabled. 
Won't that use the default `enableMoralis=true` regardless?
I might be wrong here but I'd make it false, not falsy.
```suggestion
  const useMoralis = !!useSplitExperiment("enable-moralis", true);
```